src/app/wallet/views/token_list.nim

```diff
@@ -81,3 +81,21 @@ QtObject:
     TokenRoles.Decimals.int:"decimals",
     TokenRoles.IsCustom.int:"isCustom"}.toTable
 
+  # Resolving a ENS name
+  proc getTokenDetails*(self: TokenList, address: string) {.slot.} =
+    spawnAndSend(self, "tokenDetailsResolved") do:
+      let tkn = newErc20Contract("", getCurrentNetwork(), address.parseAddress, "", 18, false)
+
+      let decimals = tkn.tokenDecimals()
```

Comment:
You get decimals here, but above you seem to hardcode 18. Is there a way to not hardcode it above?